src/main/java/picard/cmdline/PicardCommandLine.java

```diff
@@ -258,9 +264,17 @@ private static void printUsage(final Set<Class<?>> classes, final String command
                 }
                 if (!commandListOnly) {
                     if (clazz.getSimpleName().length() >= 45) {
-                        builder.append(String.format("%s    %s    %s%s%s\n", KGRN, clazz.getSimpleName(), KCYN, property.oneLineSummary(), KNRM));
+                        builder.append(
+                                String.format("%s    %s    %s%s%s%s\n",
+                                        KGRN, clazz.getSimpleName(),
+                                        KCYN, getFeaturePrefix(clazz), property.oneLineSummary(),
+                                        KNRM));
                     } else {
-                        builder.append(String.format("%s    %-45s%s%s%s\n", KGRN, clazz.getSimpleName(), KCYN, property.oneLineSummary(), KNRM));
+                        builder.append(
+                                String.format("%s    %-45s%s%s%s\n",
+                                        KGRN, clazz.getSimpleName(),
+                                        KCYN, getFeaturePrefix(clazz), property.oneLineSummary(),
+                                        KNRM));
```

Comment:
The format string doesn't seem to match the argument list. We probably need a test that executes these code paths.

Also, I'm not sure if others have an opinion, but I think it would be better to highlight the experimental/beta prefix in red, to make it stand out more from the other tools.

Reply:
Then maybe highlight feature prefixes in yellow for example, because red used in column's title
![appliedanothercolor](https://user-images.githubusercontent.com/18378140/35609962-9ce19216-0670-11e8-8d53-b588dd51ea0f.png) What you think about this?

Reply:
I think red is fine, and is consistent with what GATK uses.

Reply:
@cmnbroad all requested changes are done, thank you!